subtraction.py

```
@@ -0,0 +1,3 @@
+a = 4
+b = 3
+print (a - b)
```

Flake8 (7.2.0)

[error] whitespace before '('

(E211)